satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,25 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		id
+
+	field	id				serial
+	field	user_id user.id	restrict
+	field	offer_id offer.id	restrict
+	field	referred_by user.id	restrict
+	field	credits_earned	int
```

Comment:
Can we name this `credits_earned_in_cent` to make it clear that we are using `cent` as the unit? same with `credits_used_in_cent`

Reply:
I noticed you added this so resolving